style.css

```diff
@@ -0,0 +1,815 @@
+}
+
+.header__menu-item {
+  height: 3.48px;
+  width: 34.7px;
+  background: black;
+  margin-bottom: 5.96px;
+}
+
+.header__language {
+  position: absolute;
+  left: 27%;
+  top: 42px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  color: #fff;
+  z-index: 2;
+}
+
+.header__vector {
+  position: absolute;
+  left: 29.5%;
+  top: 54px;
+  z-index: 2;
+}
+
+.first {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 32% 68%;
+  grid-template-rows: 900px;
+}
+
+.first__sidebar {
+  grid-column: 1;
+  background: black;
+}
+
+.first__sidebar-social {
+  display: flex;
+  justify-content: space-between;
+  width: 155px;
+  padding: 816px 51px 0;
+}
+
+.collection {
+  grid-column: 2;
+  background: yellow;
+}
+
+.step {
+  position: absolute;
+  width: 100%;
+  display: grid;
+  grid-template-columns: 549px 549px;
+  grid-template-rows: 600px;
+  justify-content: center;
+  top: 168px;
+  z-index: 2;
+  grid-column-gap: 68px;
+}
+
+.img__img {
+  height: 562px;
+}
+
+.text {
+  height: 600px;
+}
+
+.text__container {
+  padding: 50px 0;
+  letter-spacing: 2px;
+}
+
+.text__title {
+  width: 549px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 80px;
+  line-height: 82px;
+  margin-bottom: 0;
+}
+
+.text__subtitle {
+  font-family: Montserrat;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 21px;
+  letter-spacing: .5px;
+  margin-bottom: 52px;
+}
+
+.collection__next-text {
+  text-decoration: none;
+  color: black;
+  font-style: normal;
+  font-size: 13px;
+  border: 1px solid black;
+  padding: 10px 23px;
+  margin-right: 52px;
+}
+
+.collection__other-text {
+  display: inline-block;
+  text-decoration: none;
+  color: black;
+  font-size: 13px;
+  letter-spacing: 0;
+  margin-right: 14px;
+}
+
+.collection__scroll {
+  position: absolute;
+  width: 27px;
+  height: 46px;
+  left: 49%;
+  top: 815px;
+  border: 2px solid #000;
+  box-sizing: border-box;
+  border-radius: 15px;
+}
+
+.collection__scroll-elem {
+  position: absolute;
+  width: 2.45px;
+  height: 8.21px;
+	left: 50%;
+	top: 826.5px;
+	background: #000;
+	border-radius: 19px;
+}
+
+.collection__next-text:hover,
+.collection__other-text:hover {
+	opacity: .5;
+	transition: 1s ease-out .2s;
+}
+
+.vectormob  {
+	display: none;
+}
+
+@media (max-width: 1235px) {
+.first {
+	max-width: 1235px;
+	width: 100%;
+	display: grid;
+	grid-template-columns: 35.2% 64.8%;
+	grid-template-rows: 719px;
+}
+
+.header__logo {
+	font-size: 24px;
+	letter-spacing: 2px;
+	margin: 5px 41px;
+}
+
+.header__menu {
+	height: 74px;
+	width: 74px;
+}
+
+.header__menu-item {
+	height: 3px;
+	width: 28.7px;
+	margin-bottom: 4px;
+}
+
+.header__language {
+	left: 29.5%;
+	top: 34px;
+	font-size: 16px;
+}
+
+.header__vector {
+	left: 32.5%;
+	top: 42px;
+}
+
+.img__img {
+	width: 439px;
+	height: 450px;
+}
+
+.step {
+	position: absolute;
+	width: 100%;
+	display: grid;
+	grid-template-columns: 439px 439px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5598887)